src/mono/wasm/runtime/library_mono.js

```diff
@@ -536,109 +535,371 @@ var MonoSupportLib = {
 			Module.ccall ('mono_wasm_load_profiler_coverage', null, ['string'], [arg]);
 		},
 
-		mono_load_runtime_and_bcl: function (vfs_prefix, deploy_prefix, enable_debugging, file_list, loaded_cb, fetch_file_cb) {
```

Comment:
This is one of the two exported functions you need to update the exports and / or keep this api for consumers even if it it implented in terms of _args

Reply:
will it actually work to keep the old API? they won't have any ICU data archives loaded if they use it

Reply:
you can call `MONO.mono_wasm_setenv ("DOTNET_SYSTEM_GLOBALIZATION_INVARIANT", "1")` for that case

Reply:
we don't strictly need to keep this api but the exports should be fixed either way